core/startup/config_file_watcher.go

```diff
@@ -93,7 +95,14 @@ func WatchConfigDirectory(configDir string, appConfig *config.ApplicationConfig)
 	// Add a path.
 	err = configWatcher.Add(configDir)
 	if err != nil {
-		return ret, fmt.Errorf("unable to establish watch on the LocalAI Configuration Directory: %+v", err)
+		return ret, fmt.Errorf("unable to establish watch on the LocalAI Configuration Directory: %w", err)
+	}
+
+	for name, watchFn := range CONFIG_FILE_UPDATES {
+		err := watchFn(configDir, appConfig)
+		if err != nil {
+			log.Warn().Msgf("Error during initial read of %q, %+v - non fatal during startup", name, err)

```

Comment:
this should be something like `log.Error().Err(err).Str("filename", name).Msg("could not process file")`